controllers/provider-aws/example/Infrastructure.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Infrastructure
+metadata:
+  name: infrastructure
+spec:
+  type: aws
+  providerConfig:
+    apiVersion: aws.provider.extensions.gardener.cloud/v1alpha1
+    kind: InfrastructureConfig
+    networks:
+      nodes: 10.250.0.0/16
+      pods: 100.96.0.0/11
+      services: 100.64.0.0/13
+      vpc:
+        cidr: 10.250.0.0/16
+      internal:
+        - 10.250.112.0/22
+      public:
+        - 10.250.96.0/22
+      workers:
+        - 10.250.0.0/19
+  zones:
+    - eu-west-1a
+  dns:
```

Comment:
Not part of the [`spec`](https://github.com/gardener/gardener/commit/6a0d9cbb9571d0145df8303044e6e60235c4d938#diff-91f1e6ef432896c6663473dfeef7c2baR47), remove it?